src/field_theory/intermediate_field.lean

```diff
@@ -273,6 +273,9 @@ def map (f : L →ₐ[K] L') : intermediate_field K L' :=
   neg_mem' := λ x hx, (S.to_subalgebra.map f).neg_mem hx,
   .. S.to_subalgebra.map f}
 
+@[simp] lemma coe_map (f : L →ₐ[K] L') : (S.map f : set L') = f '' S := rfl
+@[simp] lemma coe_sort_map (f : L →ₐ[K] L') : ↥(S.map f) = ↥(f '' S) := rfl
```

Comment:
I don't think `coe_sort_map` is a good idea. For example, I'm pretty sure we don't have an `field (f '' S)` instance.

Reply:
```suggestion
lemma coe_sort_map (f : L →ₐ[K] L') : ↥(S.map f) = ↥(f '' S) := rfl
```